sample/build.gradle

```diff
@@ -37,8 +38,13 @@ android {
 }
 
 dependencies {
-  // implementation "jp.bucketeer:sdk-android:v0.1021.0" // For test only
-  implementation(project(":bucketeer"))
+  def USE_DEV_SDK = (properties.getProperty("sample.use_dev_sdk") ?: System.getenv("USE_DEV_SDK")) == "true"
+
+  if (USE_DEV_SDK) {
+    implementation(project(":bucketeer"))
+  } else {
+    implementation "jp.bucketeer:sdk-android:0.0.5"
```

Comment:
Just confirming, how do you control the version?

Reply:
Updated version control strategy.
If a version is specified, use the version. otherwise, use latest version.